examples/online-application-portal/Ballerina.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+org = "wso2"
+name = "online_application_portal"
+version = "0.1.0"
+
+[[dependency]]
+org="ballerinax"
+name="guidewire.insnow"
+version="0.1.0"
+repository="local"
```

Comment:
We should remove this from the `Ballerina.toml` to be able to use the central packages
```suggestion
```